### PR TITLE
add password with visibility toggle

### PR DESCRIPTION
**Pull request by Stallinator**

### CODE REVIEW

File: code/TextViewPasswordVisibility.kt [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    TextField(
+        value = text,
+        onValueChange = {
+            text = it
+        },
+        modifier = Modifier
+            .wrapContentWidth()
+            .wrapContentHeight(),
+        label = { Text(text = "Your Label") },
+        placeholder = {Text(text = "Your Placeholder/Hint")},
```
Comment (Tautellini) [inline code/TextViewPasswordVisibility.kt:16]: reformat

File: code/TextViewPasswordVisibility.kt [hunk 2]
```diff
@@ -0,0 +1,31 @@
+@Composable
+fun PasswordVisibilityToggle() {
```
Comment (Tautellini) [inline code/TextViewPasswordVisibility.kt:2]: Jfyi: It is considered a good practice, to accept a modifier parameter with an empty Modifier Default. This modifier should affect the Root Composable inside of your Element/Layout.

In this repository this is okay tho.

File: code/TextViewPasswordVisibility.kt [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        modifier = Modifier
+            .wrapContentWidth()
+            .wrapContentHeight(),
+        label = { Text(text = "Your Label") },
+        placeholder = {Text(text = "Your Placeholder/Hint")},
+        keyboardOptions = KeyboardOptions(keyboardType = KeyboardType.Password),
+        visualTransformation = if (passwordVisibility) VisualTransformation.None else PasswordVisualTransformation(),
+        trailingIcon = {
+            val image = if (passwordVisibility)
+                Icons.Filled.Visibility
```
Comment (Tautellini) [inline code/TextViewPasswordVisibility.kt:21]: reformat

File: code/TextViewPasswordVisibility.kt [hunk 4]
```diff
@@ -0,0 +1,31 @@
+@Composable
+fun PasswordVisibilityToggle() {
+
+    var text by remember { mutableStateOf(TextFieldValue("")) }
+    var passwordVisibility by remember { mutableStateOf(false) }
```
Comment (Tautellini) [inline code/TextViewPasswordVisibility.kt:5]: just a small thing: your xml example is with true.

Comment (Stallinator) [reply code/TextViewPasswordVisibility.kt:5]: password not visible, the toggle visibility is set to true

**Pull request merged**